### DISCUSSION TITLE
what is the difference between driver, disk, and volume？

### DISCUSSION THREAD

**Discussion by lfygh** [minio/minio#15910] [Q&A] [answered]
what is the difference between driver, disk, and volume？ it makes me confused in minio doc.  thanks

example:
A distributed MinIO deployment consists of 4 or more **drives/volumes** managed by one or more minio server

Depending on the configured erasure code parity, a distributed deployment with m servers and n **disks** per server can continue serving read and write operations with only m/2 servers or m*n/2 drives online and accessible.

**Comment by ravindk89** [comment, marked as answer]
Hi @lfygh - we're working on cleaning up our references here, but in general they are effectively the same.

Drives, disks, and volumes are the storage resource - whatever it might be.

Moving forward we are sticking to "Drives" as the term, as it better encompasses the variety of storage types (spinning disk, flash, hybrid, etc.)
